vllm/model_executor/models/config.py

```diff
@@ -257,17 +258,28 @@ def verify_and_update_config(cls, vllm_config: "VllmConfig") -> None:
             model_config._model_info.architecture)[0]
 
         # get mamba page size
-        mamba_page_size = MambaSpec(
-            shapes=model_cls.get_mamba_state_shape_from_config(vllm_config),
-            dtype=kv_cache_dtype,
-            block_size=model_config.max_model_len,
-        ).page_size_bytes
+        static_cache_shapes = model_cls.get_mamba_state_shape_from_config(
+            vllm_config)
+        if static_cache_shapes is not None:
+            static_cache_page_size = MambaSpec(
+                shapes=static_cache_shapes,
+                dtype=kv_cache_dtype,
+                block_size=model_config.max_model_len,
+            ).page_size_bytes
+        else:
+            static_cache_shapes = model_cls.get_conv_cache_shape_from_config(
+                vllm_config)
+            static_cache_page_size = ShortConvSpec(
+                shapes=static_cache_shapes,
+                dtype=kv_cache_dtype,
+                block_size=model_config.max_model_len,
+            ).page_size_bytes
```

Comment:
Couldn't we just call a single `model_cls.get_static_cache_shape_from_config` function and determine whether to pass the shapes returned to `MambaSpec` or `ShortConvSpec` in a simpler way?